addons/csi-external-snapshotter/controller.yaml

```diff
@@ -15,11 +15,12 @@ spec:
   replicas: 2
   selector:
     matchLabels:
-      app: snapshot-controller
-  # the snapshot controller won't be marked as ready if the v1 CRDs are unavailable
-  # in #504 the snapshot-controller will exit after around 7.5 seconds if it
-  # can't find the v1 CRDs so this value should be greater than that
-  minReadySeconds: 15
+      app.kubernetes.io/name: snapshot-controller
```

Comment:
Label selectors are immutable. If we want to change this, we need an automated migration path for existing clusters.